src/components/titre-chapitre.vue

```diff
@@ -37,6 +37,7 @@ import SendRecapButton from "@/components/buttons/send-recap-button.vue"
 import { useStore } from "@/stores/index.js"
 import { useResultsStore } from "@/stores/results.js"
 import JeDonneMonAvis from "@/components/je-donne-mon-avis.vue"
+import ABTestingService from "@/plugins/ab-testing-service.js"
 
```

Comment:
Question/suggestion : faire comme dans la vue recap-email-and-sms et faire une constante showSms pour réduire le if sur le bouton et la modal title